cmd/upgrade/cluster/cmd.go

```diff
@@ -110,6 +110,8 @@ func init() {
 			"minor release, e.g. 4.12.20 -> 4.13.2. By default only z-stream updates will be scheduled. "+
 			"This is currently supported only for Hosted Control Planes. ",
 	)
+	// Hidden for now as not supported yet
+	flags.MarkHidden("allow-minor-version-updates")
```

Comment:
Does it require this check here as well? https://github.com/openshift/rosa/pull/1459/files#diff-3f508bee7428efffa6218f9a113aa7c939e1fc6dd8e30554b969aa1ccd7cc47aR149

Reply:
if you mean to explictly block the parameter usage, I am debated.
The parameter is right now rejected by the backend, but I hesitate to block it into the CLI too, as we will reenable later. So for now I just made it hidden when specified directly. WDYT? Should we just block or completely remove for now and readd later?